appconfig/settings/tailer.q

```diff
@@ -0,0 +1,10 @@
+// Bespoke WDB config : Finance Starter Pack
+
+\d .wdb
+savedir:hsym `$getenv[`KDBWDB]		// location to save wdb data
```

Comment:
not sure if savedir is needed in the tailer as it only uses the taildb

Reply:
the wdb.q sets this value itself, so removing it doesn't change the fact that a savedir exists, the tailer needs to be configured so that it doesn't write down to a wdbhdb, but as that seems like a large task, I have yet to implement it

Reply:
At the moment the tailers still write to a wdbhdb, and if it is not defined in the settings it is made somewhere else. until the write-down to a wdbhdb is removed, I thought it better to define it for now, so as to control where this wdbhdb is created

Reply:
OK I misinterpreted the above comments, it sounded like it was defined but not used. Fair enough to leave it for now then